jetty-server/src/main/config/etc/sessions/session-cache-hash.xml

```diff
@@ -13,6 +13,7 @@
      <Set name="saveOnInactiveEvict"><Property name="jetty.session.saveOnInactiveEvict" default="false" /></Set>
      <Set name="saveOnCreate"><Property name="jetty.session.saveOnCreate" default="false" /></Set>
      <Set name="removeUnloadableSessions"><Property name="jetty.session.removeUnloadableSessions" default="false"/></Set>
+     <Set name="flushOnResponseCommit"><Property name="jetty.session.flushOnCommit" default="false"/></Set>
```

Comment:
The property is only called `flushOnCommit` but all the others have the same name of the code property, so I think this should be no exception and should be renamed to `flushOnResponseCommit`.